mvg/mvg.py

```diff
@@ -879,32 +893,42 @@ def request_population_analysis(
             Source ids.
 
         feature : str
-            name of feature to run. This feature must be of population type.
+            Name of feature to run. This feature must be of population type.
 
         parameters : dict
-            name value pairs of parameters [optional].
+            Name value pairs of parameters [optional].
 
         start_timestamp : int
-            start of analysis time window [optional].
+            Start of analysis time window [optional].
 
         end_timestamp : int
-            start of analysis time window [optional].
+            Start of analysis time window [optional].
 
         callback_url : str
-            Base URL to receieve a notification when the analysis job ends.
-            A POST reuest will be made to {callback_url}/analyses with payload
+            Base URL to receive a notification when the analysis job ends.
+            A POST request will be made to {callback_url}/analyses with payload
             that includes the request_id and request_status of the job [optional].
 
+        force: bool
+            Set to False to reuse the analysis results if it already exists, otherwise
+            re-run the analysis. Defaults to False.
```

Comment:
```suggestion
            Set to True to force a re-run on any existing analysis, otherwise existing
            analysis results will be reused if possible. Defaults to False.
```